main.go

```diff
@@ -118,19 +88,19 @@ func mainWithError() error {
 	// Create a new microkit command which manages our custom microservice.
 	var newCommand command.Command
 	{
-		commandConfig := command.DefaultConfig()
-
-		commandConfig.Logger = newLogger
-		commandConfig.ServerFactory = newServerFactory
-
-		commandConfig.Description = description
-		commandConfig.GitCommit = gitCommit
-		commandConfig.Name = name
-		commandConfig.Source = source
+		c := command.Config{
+			Logger:        logger,
+			ServerFactory: serverFactory,
+
+			Description: description,
+			GitCommit:   gitCommit,
+			Name:        name,
+			Source:      source,
```

Comment:
We can also provide version bundles to make them available for the version command of the daemon.